repo/nirepo.yml

```diff
@@ -0,0 +1,1783 @@
+          default: false
+        - in: query
+          name: ignoreImportErrors
+          type: boolean
+          required: false
+          description: A query parameter that determines whether the service should abort the request if errors occur while importing some packages. If true, the service will abort and roll back the request if an error occurs while importing a package. If false, the service will import as many packages as it can, and return an error describing which packages could not be imported.
+          default: false
+        - in: body
+          name: feeds
+          description: An array containing the URIs of remote feeds whose packages should be added to the destination feed. Appending /Packages to each URI should yield the "Packages" file for that feed.
+          schema:
+            type: object
+            properties:
+              feedUris:
+                type: array
+                items:
+                  type: string
+                  example: path/to/feed
+              feedIds:
+                type: array
+                items:
+                  type: string
+                  example: 5c62bf36781e2b1c7c57daf4
+      responses:
+        202:
+          $ref: '#/responses/JobIdResponse'
+        400:
+          $ref: '#/definitions/Error'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/feeds/{id}/update-check:
+    parameters:
+      - in: path
+        name: id
+        description: The id of this resource.
+        type: string
+        required: true
+        x-example: "5c5d49f2781e2b2cd045267b"
+    post:
+      x-ni-privilege: Read
+      x-ni-operation: checkForUpdates
+      tags: [feeds]
+      summary: Check for updates to a feed
+      description: Asynchronously checks for updates to a feed, based on the latest versions of the packages available in the package sources for that feed. Offers to upgrade packages in the feed to the latest version available, as well as add any new packages to the feed if new packages were added to the package sources. Returns the ID of a resource which can be queried to determine the status of the operation. Upon completion, the resource redirects to an array of UpdateDescriptor objects, describing the updates available for the feed. Remove any updates you don't want to apply, and then send the modified array to the "Apply updates to a feed" route below.
+      parameters:
+        - in: body
+          name: packageSources
+          description: An optional array of the ids and/or external uris of package sources which should be queried for updates. If not given, all of the feed's package sources will be checked. Package sources that are not present in the feed's array of package sources will be ignored. If you would like to add additional package sources to a feed, use the import route.
```

Comment:
Would it be improper to capitalize IDs and URIs?

Reply:
I think it would work ok to capitalize on them.